driver/bpf/fillers.h

```diff
@@ -6280,9 +6280,9 @@ FILLER(sys_dup3_x, true)
 	/*
 	 * flags
 	 */
-	val = bpf_syscall_get_argument(data, 2);
-	flags = dup3_flags_to_scap(val);
-	return bpf_push_u32_to_ring(data, flags);
+	int flags = bpf_syscall_get_argument(data, 2);
+	scap_flags = dup3_flags_to_scap(flags);
+	return bpf_push_u32_to_ring(data, scap_flags);
```

Comment:
```suggestion
	return bpf_push_u32_to_ring(data, dup3_flags_to_scap(flags));
```

Reply:
Was there a reason it wasn't like that before? Sometimes I am suspicious if something is a certain way in the old bpf probe ... Yes it would be a nice cleanup.